mcstatus/motd.py

```diff
@@ -0,0 +1,558 @@
+
+            for element in item["extra"]:
+                parsed_motd.extend(cls._parse_as_dict(element, auto_add=auto_add.copy()))
+
+        return parsed_motd
+
+    @staticmethod
+    def _parse_as_str(raw: str, *, bedrock: bool = False) -> list[ParsedMotdItem]:
+        """Parse a MOTD when it's string.
+
+        .. note:: This method returns a lot of empty strings, use ``Motd.simplify`` to remove them.
+
+        :param raw: Raw MOTD, directly from server.
+        :param bedrock: Is server Bedrock Edition?
+            Ignores ``MinecraftColor.MINECOIN_GOLD`` if it's ``False``.
+        :returns: ``ParsedMotdItem`` list, which need to be passed to ``__init__``.
+        """
+        parsed_motd: list[ParsedMotdItem] = []
+
+        split_raw = MOTD_COLORS_RE.split(raw)
+        for element in split_raw:
+            clean_element = element.lstrip("&§").lower()
+            standardized_element = element.replace("&", "§").lower()
+
+            if standardized_element == "§g" and not bedrock:
+                parsed_motd.append(element)  # minecoin_gold on java server
+                continue
+
+            if standardized_element.startswith("§"):
+                try:
+                    parsed_motd.append(MinecraftColor(clean_element))
+                except ValueError:
+                    try:
+                        parsed_motd.append(Formatting(clean_element))
+                    except ValueError:
+                        # just a text
+                        parsed_motd.append(element)
+            else:
+                parsed_motd.append(element)
+
+        return parsed_motd
+
+    def simplify(self) -> Self:
+        """Simplify MOTD by removing unused formatting, unaffected colors etc.
+
+        :returns: New instance with simplified ``parsed`` field.
+        """
+        parsed = self.parsed.copy()
+        simplifies_class = MotdSimplifies(parsed)
+
+        old_motd = []
+        while old_motd != parsed and parsed:
+            for enumeration, item in enumerate(parsed):
+                old_motd = parsed.copy()
+                for hook in simplifies_class.get_simplifies():
+                    hook(item, enumeration)
+
+        return Motd(parsed, self.bedrock)
+
+    @property
+    def plain(self) -> str:
+        """Get plain text from MOTD.
+
+        :returns: Plain text.
+        """
+        return "".join(str(e) for e in self.parsed if isinstance(e, str))
+
+    @property
+    def minecraft(self) -> str:
+        """Getter for variant of a MOTD with Minecraft colors.
+
+        :returns: Minecraft variant of MOTD.
+
+        .. note:: This will always use "§", even if in original MOTD everywhere was "&".
+        """
+        result = ""
+        for element in self.parsed:
+            if isinstance(element, str):
+                result += element
+                continue
+            if isinstance(element, WebColor) or isinstance(element, TranslationTag):
```

Comment:
`isinstance` can check multiple things at once, no need for an `or` here
```suggestion
            if isinstance(element, (WebColor, TranslationTag)):
```

Reply:
Fixed [here](https://github.com/py-mine/mcstatus/compare/26690ce403010ab0af019cf2230d246e2920602e..ec2cacf3a1ee18ca0432a5c5e54c80ee6dbc3fcb), thanks!